Sources/NIOWebSocket/WebSocketErrorCodes.swift

```diff
@@ -2,7 +2,7 @@
 //
 // This source file is part of the SwiftNIO open source project
 //
-// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Copyright (c) 2017-2032 Apple Inc. and the SwiftNIO project authors
```

Comment:
```suggestion
// Copyright (c) 2017-2021 Apple Inc. and the SwiftNIO project authors
```

Reply:
Wait, I didn't write this in the future?